src/udev/udev-builtin-net_id.c

```diff
@@ -231,9 +232,9 @@ static int dev_pci_onboard(struct udev_device *dev, struct netnames *names) {
         if (!attr)
                 return -ENOENT;
 
-        idx = strtoul(attr, NULL, 0);
-        if (idx <= 0)
-                return -EINVAL;
+        ret = safe_atolu(attr, &idx);
+        if (ret)
```

Comment:
please test with "ret < 0" for errors, simply to keep this in style with the rest of the codebase.

Reply:
OK ,fixed.